package-lock.json

```diff
@@ -0,0 +1,18397 @@
+{
```

Comment:
can we please not add this? if this is important, please put it into its own PR with a description why.